README.md

```diff
@@ -45,8 +45,8 @@ With Gradle from repo.spring.io:
     }
 
     dependencies {
-      compile "io.projectreactor.kafka:reactor-kafka:1.3.23"
-      // compile "io.projectreactor.kafka:reactor-kafka:1.3.24-SNAPSHOT"
+      // compile "io.projectreactor.kafka:reactor-kafka:1.4.1-SNAPSHOT"
+      compile "io.projectreactor.kafka:reactor-kafka:1.4.0"
```

Comment:
I think that for the stable release we have to use `1.3.23` as we do not have any other, while for the snapshot we have to use `1.4.0-SNAPSHOT`

```suggestion
      // compile "io.projectreactor.kafka:reactor-kafka:1.4.0-SNAPSHOT"
      compile "io.projectreactor.kafka:reactor-kafka:1.3.23"
```